_data/simple-icons.json

```
@@ -5222,6 +5222,11 @@
             "source": "https://en.wikipedia.org/wiki/File:Hyundai_Motor_Company_logo.svg",
             "guidelines": "https://www.hyundai.pl/fileadmin/user_upload/media/logo/201607_HYU_Guideline_ENG_small.pdf"
         },
+        {
+            "title": "h&m",
+            "hex": "e50010",
+            "source": "https://www.hm.com/entrance.ahtml",
```

```suggestion
            "source": "https://www.hm.com"
```
entrance.ahtml unnecessary
comma cannot be added on last key value